meta-mbl-bsp-common/recipes-bsp/atf/atf.inc

```diff
@@ -14,10 +14,13 @@ SRC_URI = " \
     git://github.com/ARMmbed/mbedtls.git;protocol=https;branch=development;name=mbedtls;destsuffix=mbedtls \
 "
 
+# MBL_ATF_VERSION should be updated to match version pointed to by SRCREV
```

Comment:
2 observations:

1. The atf build directory should be under the machine's WORKDIR, and for Rpi it is under the core architecture (e.g. `cortexa7t2hf-neon-vfpv4-oe-linux-gnueabi`);
2. We should set the atf package version properly.

My suggestion is to add the following lines to this file:

```
PACKAGE_ARCH = "${MACHINE_ARCH}"
PV = "${MBL_ATF_VERSION}+git${SRCPV}"
```

Reply:
The WORKDIR for rpi3 is not affected by my patch but, I can look at it since you've raised it.
The package version I think is an easy change.